### PR TITLE
add page for PR writing and reviewing

### PR DESCRIPTION
**Pull request by marypelletier**
#### In this PR
- Add page about PR writing and reviewing
- Add link for it to the readme
- Add link in a11y doc for how to enable keyboard tabbing in FF for Mac

### CODE REVIEW

File: ux-dev/pull-requests.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# UXD PR Review Checklist
+
+## Before the PR
+Ensure that your changes are:
++ Accurate to design: 
+   - Follow the guidelines in the [https://github.com/DockYard/styleguides/blob/master/ux-dev/pixel-perfection.md](Pixel Perfection) guide
```
Comment (crowjm) [inline ux-dev/pull-requests.md]: For some reason the link syntax here isn't rendering properly. 

Additionally, I would lower case "pixel perfection" for consistency

File: ux-dev/pull-requests.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+   - Check that copy is accurate with no spelling errors
+   - Complies with any applicable style guide
++ Responsive: Check all target viewport sizes based designs and target devices
++ Accessible:
+  - Follow guidelines in the [https://github.com/DockYard/styleguides/blob/master/ux-dev/accessibility-a11y.md](accessibility) guide
```
Comment (crowjm) [inline ux-dev/pull-requests.md]: This link syntax is also not rendering correctly.

File: ux-dev/pull-requests.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  - Test for keyboard usability 
+  - Test with screen reader
++ Usable: check all related interactive elements for expected functionality, flows, and states
++ Performant: use the browser's performance monitor panel to assess the impact of your changes
++ Compatible:
+  - Check all browsers in support target
+  - Use an [https://validator.w3.org/](HTML validator)
```
Comment (crowjm) [inline ux-dev/pull-requests.md]: Link syntax not rendering properly.

File: ux-dev/pull-requests.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  - Optional for browsers/devices have different compatibility needs:
+    + progressive enhancements
+    + fallbacks
+    + polyfills
+    + @supports
```
Comment (crowjm) [inline ux-dev/pull-requests.md]: Thoughts on wrapping "@supports" in backticks?

File: ux-dev/pull-requests.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+Here are some simple ways to make your visuals of the changes be as helpful as possible:
+
++ Always include design specs for reference and comparison
++ Screen shots of the app changes:
+  - Capture before and after
+  - Consider including markup on the screen shots for clarity
++ Gifs of the app changes:
+  - Capture before and after
+  - Ideal for PRs where the changes include multiple component states, animations/transitions, or interactions
+  - [https://www.cockos.com/licecap/](LICEcap) is a free gif recording tool
```
Comment (crowjm) [inline ux-dev/pull-requests.md]: Link syntax isn't rendering properly.

**Comment by crowjm** [comment]
@marypelletier I also merged a new PR in, so be sure to pull down to make sure you have that.

**Comment by marypelletier** [comment]
@crowjm thank you! I just pushed changes to fix the links and add those backticks.

File: ux-dev/accessibility-a11y.md [hunk 6]
```diff
@@ -18,6 +18,8 @@ The following keyboard commands are helpful:
 - `Enter` to activate links and buttons
 - `Space` to activate checkboxes and buttons
 
+You may need to enable tab navigation in Firefox for Mac. There is more information [https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility#Using_native_keyboard_accessibility](here) at the first bullet point of the "Using native keyboard accessibility" section.
```
Comment (crowjm) [inline ux-dev/accessibility-a11y.md]: @marypelletier One more link that isn't rendering properly. After this is fixed it'll be good to merge.

**Pull request merged**